### PR TITLE
Implement pack-plugin.php

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: pack-plugin.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+$directory = __DIR__ . '/plugins/' . $plugin . '/src';
+
+if (! $plugin || ! preg_match('~^(?:[a-z-_]++)$~', $plugin) || ! is_dir($directory)) {
+    echo 'Plugin name was not specified or is invalid.' . PHP_EOL;
+    exit(2);
+}
+
+chdir($directory);
+
+if (file_exists($directory . '/composer.json')) {
```
Comment (keksa) [inline pack-plugin.php]: extend the test for PRs to check whether there is `compose.lock` file if `composer.json` is present

**Comment by Piskvor** [comment]
Do we want to lint PHP? (Not necessary, but could be useful - either in the packing script, or in validate.php|Travis)

File: pack-plugin.php (hunk 1 above)
Comment (Piskvor) [reply pack-plugin.php]: Perhaps to https://github.com/Ubiquiti-App/UCRM-plugins/pull/81

Comment (Piskvor) [reply pack-plugin.php]: https://github.com/Ubiquiti-App/UCRM-plugins/pull/81/commits/946522f3e688b4d486b4006691223b5eba97cb22

**Comment by Piskvor** [comment]
> Do we want to lint PHP? (Not necessary, but could be useful - either in the packing script, or in validate.php|Travis)

https://github.com/Ubiquiti-App/UCRM-plugins/pull/81/commits/5718545b5ceaf8a4fa642fe28d2f881fd59ba047

**Pull request merged**